opendcs-rest-api/src/main/java/org/opendcs/odcsapi/dao/ApiDaoBase.java

```diff
@@ -15,46 +15,64 @@
 
 package org.opendcs.odcsapi.dao;
 
-import org.opendcs.odcsapi.hydrojson.DbInterface;
-import org.opendcs.odcsapi.util.ApiConstants;
-
-import decodes.sql.DbKey;
-
-import java.util.logging.Logger;
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.logging.Logger;
 
-public class ApiDaoBase
-	implements ApiDaiBase
+import decodes.sql.SqlDatabaseIO;
+import org.opendcs.odcsapi.hydrojson.DbInterface;
+import org.opendcs.odcsapi.util.ApiConstants;
+
+public class ApiDaoBase implements ApiDaiBase
 {
+	private final SqlDatabaseIO dbio;
```

Comment:
This one should also be the OpenDcsDatabase class. It'll have the getDao method and an accessor to the "Database" instance (which has the DbIo if you need it.)

Reply:
oh yeah, I need to revert changes in this PR and just deprecate the whole class